packages/ubuntu_bootstrap/lib/pages/install/install_model.dart

```diff
@@ -172,6 +175,10 @@ class InstallModel extends SafeChangeNotifier {
     super.dispose();
   }
 
+  var _provisioningMode = ProvisioningMode.standard;
```

Comment:
This might be one of the cases where `late final` might actually be better (I'd consider a runtime error 'better' than returning the wrong provisioning mode here). 
@spydon any thoughts?

Reply:
Changed to late final. Opinions?